portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,47 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// the indexes recording the picture 
+// the slides in gallery.html is showing
+var slide1 = 0;
+var slide2 = 0;
+
 /**
- * Adds a random greeting to the page.
+ * Controls(flips) both slideshow in gallery.html
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function flip(/* num of pages to flip= */ num, 
```

Comment:
Remove comments before parameters. We only use this format when we pass a constant to a function and the comment would read the parameter name.